src/HTTPServer.cpp

```diff
@@ -28,11 +28,10 @@ int HTTPServer::run()
 		logger.log(INFO, "Server started");
 		while (true)
```

Comment:
Does this belong in the Try/Catch block.

Reply:
made a seperate try catch block which doesn't stop the server